addon/utils/scroll-helpers.js

```diff
@@ -55,26 +126,46 @@ export function distributeScroll(deltaX, deltaY, element, container) {
     deltaYPositive: scrollTopMax - element.scrollTop
   };
 
-  element.scrollLeft = element.scrollLeft + deltaX;
-  element.scrollTop = element.scrollTop + deltaY;
+  const elementStyle = window.getComputedStyle(element);
```

Comment:
I'm concerned that this is called for every scroll event. If that is the case, could it be cached?

Reply:
It can change between every single turn of the event loop since we yield the thread. As a result the only way to ensure we get the right value is to ask the DOM again.